torch/distributed/_shard/sharded_tensor/_ops/math_ops.py

```diff
@@ -83,21 +83,25 @@ def binary_math_op(types, args=(), kwargs=None, pg=None):
     # add
     torch.add,
     Tensor.add,
+    Tensor.add_,
```

Comment:
is this working? I think we need to add a `add_` method to ShardedTensor class and dispatch to here to make it really work

Reply:
Hmm, that's wired because the unit test does pass.